### PR TITLE
Update graphic modes

### PR DESCRIPTION
**Pull request by ahoneybun**
This adds the commands for running applications though the NVIDIA GPU while in Hybrid mode.

### CODE REVIEW

File: _articles/graphics-switch-pop.md [hunk 1]
```diff
@@ -42,6 +45,18 @@ Hybrid graphics mode uses both the integrated Intel GPU and the discrete NVIDIA
 Applications will use the integrated GPU unless explicitly requested to use the
 discrete GPU.
 
+Vulkan applications must be launched with the following command before the application name to be rendered on the dGPU (NVIDIA):
+
+```
+__NV_PRIME_RENDER_OFFLOAD=1 *application*
```
Comment (crawfxrd) [inline _articles/graphics-switch-pop.md]: Use angle brackets for required arguments: `<application>`

File: _articles/graphics-switch-pop.md [hunk 2]
```diff
@@ -42,6 +45,18 @@ Hybrid graphics mode uses both the integrated Intel GPU and the discrete NVIDIA
...
+```
+
+GLX applications much be launcher with this command before the application to be rendered on the dGPU (NVIDIA):
```
Comment (crawfxrd) [inline _articles/graphics-switch-pop.md]: A right click context menu item is also available in newer versions of GNOME Shell to launch applications on the dGPU. (Did it make it into 3.35.91?)

Comment (ahoneybun) [reply _articles/graphics-switch-pop.md]: I'll need to get a machine that can do that to test.

File: _articles/graphics-switch-pop.md [hunk 3]
```diff
@@ -42,6 +45,18 @@ Hybrid graphics mode uses both the integrated Intel GPU and the discrete NVIDIA
 Applications will use the integrated GPU unless explicitly requested to use the
 discrete GPU.
 
+Vulkan applications must be launched with the following command before the application name to be rendered on the dGPU (NVIDIA):
```
Comment (crawfxrd) [inline _articles/graphics-switch-pop.md]: "before the application [name]" can probably be dropped.

File: _articles/graphics-switch-pop.md [hunk 4]
```diff
@@ -42,6 +45,18 @@ Hybrid graphics mode uses both the integrated Intel GPU and the discrete NVIDIA
...
+
+```
+__NV_PRIME_RENDER_OFFLOAD=1 <application>
+```
+
+GLX applications much be launcher with this command before the application to be rendered on the dGPU (NVIDIA):
```
Comment (crawfxrd) [inline _articles/graphics-switch-pop.md]: `launcher` -> `launched`

File: _articles/graphics-switch-pop.md (hunk 2 above)
Comment (ahoneybun) [reply _articles/graphics-switch-pop.md]: @crawfxrd I just upgraded to GNOME 3.35.91 and I don't see the option in Pop.

File: _articles/graphics-switch-pop.md [hunk 5]
```diff
@@ -42,6 +45,18 @@ Hybrid graphics mode uses both the integrated Intel GPU and the discrete NVIDIA
 Applications will use the integrated GPU unless explicitly requested to use the
 discrete GPU.
 
+Vulkan applications must be launched with the following command to be rendered on the dGPU (NVIDIA):
+
+```
+__NV_PRIME_RENDER_OFFLOAD=1 <application>
+```
+
+GLX applications much be launched with this command before the application to be rendered on the dGPU (NVIDIA):
```
Comment (crawfxrd) [inline _articles/graphics-switch-pop.md]: "before the application" can be dropped.

Comment (storyaddict) [inline _articles/graphics-switch-pop.md:54]: Typo, and removing the "before the application."

"GLX applications must be launched with this command to be rendered on the dDPU (NVIDIA):"

**Comment by crawfxrd** [comment]
"Intel" mode is in the process of being renamed to "Integrated" mode.

**Pull request merged**